docs/_build/html/web3-eth.html

```diff
@@ -657,7 +657,7 @@ <h2>getStorageAt<a class="headerlink" href="#getstorageat" title="Permalink to t
 <h3>Parameters<a class="headerlink" href="#id35" title="Permalink to this headline">¶</a></h3>
 <ol class="arabic simple">
 <li><code class="docutils literal"><span class="pre">String</span></code> - The address to get the storage from.</li>
-<li><code class="docutils literal"><span class="pre">Number</span></code> - The index position of the storage.</li>
+<li><code class="docutils literal"><span class="pre">Number|String</span></code> - The index position of the storage.</li>
```

Comment:
The build file can get removed because ``readthedocs`` isn't using the provided build it does run the ``docs`` build pipeline on its own.